assertj-guava has version range dependency

Building jdbi-3.6.0 on MacOS 10.14.3 using Maven 3.6.0.

Used command `mvn clean install`

Got Error: 
```
[WARNING] Potential problems found in Dependency Management 
[INFO] 
[INFO] --- maven-dependency-versions-check-plugin:2.0.4:check (default) @ jdbi3-guava ---
[INFO] Checking dependency versions
[ERROR] Found a problem with the direct dependency org.assertj:assertj-core of the current project
  Expected version is 3.11.1
  Resolved version is 3.11.1
  Version 3.12.1 was expected by artifact: org.assertj:assertj-guava
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for jdbi3 Parent 3.6.0:
[INFO] 
[INFO] jdbi3 Policy ....................................... SUCCESS [ 31.829 s]
[INFO] jdbi3 Parent ....................................... SUCCESS [ 16.495 s]
[INFO] jdbi3 Core ......................................... SUCCESS [01:11 min]
[INFO] jdbi3 Test Helpers ................................. SUCCESS [ 15.473 s]
[INFO] jdbi3 Guava Integration ............................ FAILURE [ 10.215 s]
[INFO] jdbi3 SqlObject .................................... SKIPPED
[INFO] jdbi3 String Template 4 integration ................ SKIPPED
[INFO] jdbi3 Apache Commons Text integration .............. SKIPPED
[INFO] jdbi3 Freemarker integration ....................... SKIPPED
[INFO] jdbi3 json ......................................... SKIPPED
[INFO] jdbi3 Postgres ..................................... SKIPPED
[INFO] jdbi3 gson2 ........................................ SKIPPED
[INFO] jdbi3 jackson2 ..................................... SKIPPED
[INFO] jdbi3 Joda-Time Integration ........................ SKIPPED
[INFO] jdbi3 JPA Integration .............................. SKIPPED
[INFO] jdbi3 Spring ....................................... SKIPPED
[INFO] jdbi3 sqlite ....................................... SKIPPED
[INFO] jdbi3 Vavr Integration ............................. SKIPPED
[INFO] jdbi3 Kotlin plugin ................................ SKIPPED
[INFO] jdbi3 SqlObject Kotlin plugin ...................... SKIPPED
[INFO] jdbi3 documentation ................................ SKIPPED
[INFO] jdbi3 BOM .......................................... SKIPPED
[INFO] jdbi3 noparameters tests ........................... SKIPPED
[INFO] Jdbi JMH benchmarks ................................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:28 min
[INFO] Finished at: 2019-03-14T21:20:54-06:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.ning.maven.plugins:maven-dependency-versions-check-plugin:2.0.4:check (default) on project jdbi3-guava: Found dependency version conflicts -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :jdbi3-guava
```

I also tried building all the other version 3 releases. Each failed.

--- a/pom.xml
+++ b/pom.xml
@@ -371,7 +371,7 @@
             <dependency>
                 <groupId>org.assertj</groupId>
                 <artifactId>assertj-guava</artifactId>
-                <version>3.2.0</version>
+                <version>3.2.1</version>
             </dependency>
 
             <dependency>

